frontends/web/src/routes/account/send/components/inputs/receiver-address-input.tsx

```diff
@@ -0,0 +1,78 @@
+import { SyntheticEvent, useContext } from 'react';
+import { useTranslation } from 'react-i18next';
+import { debug } from '../../../../../utils/env';
+import { getReceiveAddressList } from '../../../../../api/account';
+import DarkModeContext from '../../../../../contexts/DarkmodeContext';
+import { Input } from '../../../../../components/forms';
+import qrcodeIconDark from '../../../../../assets/icons/qrcode-dark.png';
+import qrcodeIconLight from '../../../../../assets/icons/qrcode-light.png';
+import style from '../../send.module.css';
+
+type TToggleScanQRButtonProps = {
+    onClick: () => void;
+}
+
+type TReceiverAddressInputProps = {
+    accountCode?: string;
+    addressError?: string;
+    hasCamera: boolean;
+    onClickScanQRButton: () => void;
+    onClickSendToSelfButton: (e: SyntheticEvent, receiveAddress: string) => void;
+    onInputChange: (e: SyntheticEvent) => void;
+    recipientAddress: string;
+}
+
+const ScanQRButton = ({ onClick }: TToggleScanQRButtonProps) => {
+  const { isDarkMode } = useContext(DarkModeContext);
+  return (
+    <button onClick={onClick} className={style.qrButton}>
+      <img src={isDarkMode ? qrcodeIconLight : qrcodeIconDark} />
```

Comment:
Once we have the svg versions of this icon (light&dark), we can do the usual and import from `icon.tsx` (instead of putting .png into `img` directly like what we're doing here).